modules/latex/latex.js

```diff
@@ -0,0 +1,19 @@
+( function( $ ) {
+	'use strict';
+	$( document.body ).on( 'post-load', function() {
+		// New posts have been added to the page. Re-typeset MathJax if necessary.
+		if ( typeof MathJax !== 'undefined' ) {
+			$( '.infinite-wrap' ).not( '.mathjax-typeset' ).each( function() {
+				var wrap = this;
+
+				MathJax.Hub.Queue( [
+					'Typeset',
+					MathJax.Hub,
+					wrap
+				] );
+
+				$( wrap ).addClass( 'mathjax-typeset' );
+			} );
```

Comment:
If this can't be solved I don't know if it's worth switching to MathJax. It's too unreliable with infinite scroll. I've spent hours and hours trying various things and nothing works 100% of the time.